src/qt/res/css/general.css

```diff
@@ -68,7 +68,7 @@ QDialog {
 }
 
 .QFrame {
-    background-color: transparent;
+    background-color: #00000000;
```

Comment:
can you explain the reason for this?

Reply:
Just that relevant colors are all in the same `#aarrggbb` format and that its catched by `contrib/devtools/update-css-files.py`. There is still the named color `red` but thats only a placeholder to show an undefined behaviour in the UI. So if there is a bright red somewhere in the UI it means the related color in `general.css` isn't overwritten in the theme file. I want to write some UI development style guidelines anytime later where i will explain such thoughts i had.